azure-functions/acbs-function/apim-mdm.js

```diff
@@ -0,0 +1,36 @@
+require('dotenv').config();
+const axios = require('axios');
+
+const headers = {
+  [String(process.env.APIM_MDM_KEY)]: process.env.APIM_MDM_VALUE,
+};
+
+/**
+ * APIM MDM `GET` generic endpoint invocation call
+ * @param {String} endpoint Endpoint with provided query parameters
+ * @returns Response object
+ */
+const getAPI = async (endpoint) => {
+  const response = await axios({
+    method: 'get',
+    url: `${process.env.APIM_MDM_URL}${endpoint}`,
```

Comment:
This used to be `${process.env.MULESOFT_API_UKEF_MDM_EA_URL}/${type}` which adds a `/` between the environment variable and the input arg, but the new code isn't adding the `/`. Have we added the slash to the end of the `APIM_MDM_URL` environment variable instead?

Reply:
There are a few other places in the code where we've used `process.env.APIM_MDM_URL` without adding our own slash at the end - I've not commented on those to save you from the repeated comment, but if we do need to change it then there are a few other places to change too

Reply:
We have indeed added `/` to the env variable `APIM_MDM_URL`.